[2025-04-06 11:58:09,011: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 11:58:09,014: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 11:58:09,016: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 11:58:09,018: INFO: common: Created direectories at: artifacts]
[2025-04-06 11:58:09,019: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 11:58:09,722: INFO: 1564805586: artifacts/data_ingestion/data.zip download! With following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7B68:75D66:40D367:A60829:67F2124F
Accept-Ranges: bytes
Date: Sun, 06 Apr 2025 06:28:08 GMT
Via: 1.1 varnish
X-Served-By: cache-del21730-DEL
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1743920889.672225,VS0,VE294
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 8a44f837148698f608264e1edf2c6cfe4733a2b6
Expires: Sun, 06 Apr 2025 06:33:08 GMT
Source-Age: 0

]
[2025-04-06 12:02:28,321: INFO: main: Welcome to our cutome logging data science project]
[2025-04-06 12:02:28,337: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-06 12:02:28,337: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 12:02:28,337: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 12:02:28,337: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 12:02:28,337: INFO: common: Created direectories at: artifacts]
[2025-04-06 12:02:28,337: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 12:02:28,337: INFO: data_ingestion: File already exists]
[2025-04-06 12:02:28,346: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 15:12:25,020: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:12:25,042: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:12:25,064: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:12:25,080: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:12:25,082: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 15:13:42,214: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:13:42,232: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:13:42,236: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:13:42,238: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:13:42,239: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 15:18:28,100: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:18:28,102: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:18:28,105: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:18:28,108: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:18:28,108: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 15:19:26,683: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:19:26,684: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:19:26,687: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:19:26,688: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:19:26,689: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-06 15:25:18,068: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:25:18,071: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:25:18,081: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:25:18,082: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:25:18,082: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-06 15:28:31,725: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:28:31,727: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:28:31,732: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:28:31,733: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:28:31,734: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-06 15:53:23,781: INFO: main: Welcome to our cutome logging data science project]
[2025-04-06 15:53:23,781: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-06 15:53:23,781: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:53:23,781: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:53:23,786: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:53:23,786: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:53:23,786: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 15:53:23,786: INFO: data_ingestion: File already exists]
[2025-04-06 15:53:23,827: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 15:53:23,827: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-06 15:53:23,827: ERROR: main: 'DataValidationTrainingPipeline' object has no attribute 'initiate_data_validation']
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\main.py", line 24, in <module>
    data_validation.initiate_data_validation()
AttributeError: 'DataValidationTrainingPipeline' object has no attribute 'initiate_data_validation'
[2025-04-06 15:58:00,370: INFO: main: Welcome to our cutome logging data science project]
[2025-04-06 15:58:00,370: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-06 15:58:00,370: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 15:58:00,370: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 15:58:00,378: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 15:58:00,379: INFO: common: Created direectories at: artifacts]
[2025-04-06 15:58:00,379: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 15:58:00,379: INFO: data_ingestion: File already exists]
[2025-04-06 15:58:00,382: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 15:58:00,382: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-06 15:58:00,382: ERROR: main: 'DataValidationTrainingPipeline' object has no attribute 'initiate_data_validation']
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\main.py", line 24, in <module>
    data_ingestion.initiate_data_validation()
AttributeError: 'DataValidationTrainingPipeline' object has no attribute 'initiate_data_validation'
[2025-04-06 16:00:00,625: INFO: main: Welcome to our cutome logging data science project]
[2025-04-06 16:00:00,625: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-06 16:00:00,627: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 16:00:00,628: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 16:00:00,630: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 16:00:00,630: INFO: common: Created direectories at: artifacts]
[2025-04-06 16:00:00,630: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 16:00:00,630: INFO: data_ingestion: File already exists]
[2025-04-06 16:00:00,633: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 16:00:00,633: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-06 16:00:00,636: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 16:00:00,637: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 16:00:00,637: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 16:00:00,637: INFO: common: Created direectories at: artifacts]
[2025-04-06 16:00:00,637: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-06 16:00:00,656: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 16:48:24,551: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 16:48:24,554: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 16:48:24,558: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 16:48:24,559: INFO: common: Created direectories at: artifacts]
[2025-04-06 16:48:24,563: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-06 16:50:29,326: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 16:50:29,326: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 16:50:29,326: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 16:50:29,326: INFO: common: Created direectories at: artifacts]
[2025-04-06 16:50:29,337: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-06 16:53:48,281: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 16:53:48,283: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 16:53:48,283: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 16:53:48,283: INFO: common: Created direectories at: artifacts]
[2025-04-06 16:53:48,283: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-06 16:53:48,395: INFO: 2186127381: Splited data into training and test sets]
[2025-04-06 16:53:48,395: INFO: 2186127381: (1199, 12)]
[2025-04-06 16:53:48,395: INFO: 2186127381: (400, 12)]
[2025-04-06 17:20:02,601: INFO: main: Welcome to our cutome logging data science project]
[2025-04-06 17:20:02,601: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-06 17:20:02,603: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 17:20:02,604: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 17:20:02,606: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 17:20:02,607: INFO: common: Created direectories at: artifacts]
[2025-04-06 17:20:02,607: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 17:20:02,607: INFO: data_ingestion: File already exists]
[2025-04-06 17:20:02,618: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 17:20:02,618: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-06 17:20:02,618: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 17:20:02,618: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 17:20:02,627: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 17:20:02,628: INFO: common: Created direectories at: artifacts]
[2025-04-06 17:20:02,628: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-06 17:20:02,642: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 17:20:02,642: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-06 17:20:02,651: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 17:21:08,608: INFO: main: Welcome to our cutome logging data science project]
[2025-04-06 17:21:08,608: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-06 17:21:08,608: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 17:21:08,608: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 17:21:08,615: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 17:21:08,615: INFO: common: Created direectories at: artifacts]
[2025-04-06 17:21:08,615: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-06 17:21:08,618: INFO: data_ingestion: File already exists]
[2025-04-06 17:21:08,621: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 17:21:08,622: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-06 17:21:08,625: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 17:21:08,626: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 17:21:08,629: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 17:21:08,629: INFO: common: Created direectories at: artifacts]
[2025-04-06 17:21:08,630: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-06 17:21:08,644: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-06 17:21:08,644: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-06 17:21:08,648: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-06 17:21:08,649: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-06 17:21:08,651: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-06 17:21:08,651: INFO: common: Created direectories at: artifacts]
[2025-04-06 17:21:08,652: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-06 17:21:08,672: INFO: data_transformation: Splited data into training and test sets]
[2025-04-06 17:21:08,672: INFO: data_transformation: (1199, 12)]
[2025-04-06 17:21:08,672: INFO: data_transformation: (400, 12)]
[2025-04-06 17:21:08,673: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-08 09:31:24,947: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 09:31:24,971: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 09:31:24,974: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 09:31:24,974: INFO: common: Created direectories at: artifacts]
[2025-04-08 09:34:09,332: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 09:34:09,334: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 09:34:09,334: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 09:34:09,334: INFO: common: Created direectories at: artifacts]
[2025-04-08 09:38:55,245: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 09:38:55,249: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 09:38:55,254: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 09:38:55,256: INFO: common: Created direectories at: artifacts]
[2025-04-08 09:39:29,658: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 09:39:29,661: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 09:39:29,662: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 09:39:29,662: INFO: common: Created direectories at: artifacts]
[2025-04-08 09:42:31,358: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 09:42:31,359: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 09:42:31,366: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 09:42:31,368: INFO: common: Created direectories at: artifacts]
[2025-04-08 18:23:53,453: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 18:23:53,471: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 18:23:53,479: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 18:23:53,481: INFO: common: Created direectories at: artifacts]
[2025-04-08 18:27:09,504: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 18:27:09,506: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 18:27:09,509: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 18:27:09,510: INFO: common: Created direectories at: artifacts]
[2025-04-08 18:31:59,565: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 18:31:59,568: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 18:31:59,572: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 18:31:59,574: INFO: common: Created direectories at: artifacts]
[2025-04-08 18:31:59,576: INFO: common: Created direectories at: artifacts]
[2025-04-08 18:32:57,955: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 18:32:57,957: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 18:32:57,961: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 18:32:57,963: INFO: common: Created direectories at: artifacts]
[2025-04-08 18:32:57,976: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-08 19:01:10,064: INFO: main: Welcome to our cutome logging data science project]
[2025-04-08 19:01:10,064: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-08 19:01:10,064: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 19:01:10,064: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 19:01:10,064: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 19:01:10,078: INFO: common: Created direectories at: artifacts]
[2025-04-08 19:01:10,079: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-08 19:01:10,079: INFO: data_ingestion: File already exists]
[2025-04-08 19:01:10,162: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-08 19:01:10,162: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-08 19:01:10,162: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 19:01:10,162: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 19:01:10,162: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 19:01:10,162: INFO: common: Created direectories at: artifacts]
[2025-04-08 19:01:10,162: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-08 19:01:10,183: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-08 19:01:10,183: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-08 19:01:10,197: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 19:01:10,198: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 19:01:10,200: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 19:01:10,201: INFO: common: Created direectories at: artifacts]
[2025-04-08 19:01:10,201: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-08 19:01:10,285: INFO: data_transformation: Splited data into training and test sets]
[2025-04-08 19:01:10,285: INFO: data_transformation: (1199, 12)]
[2025-04-08 19:01:10,285: INFO: data_transformation: (400, 12)]
[2025-04-08 19:01:10,286: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-08 19:01:10,286: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-08 19:01:10,289: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 19:01:10,290: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 19:01:10,292: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 19:01:10,293: INFO: common: Created direectories at: artifacts]
[2025-04-08 19:01:10,293: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-08 19:01:10,322: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-08 22:29:30,087: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:29:30,091: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:29:30,096: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:29:30,098: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:29:54,104: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:29:54,107: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:29:54,110: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:29:54,112: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:36:10,465: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:36:10,468: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:36:10,472: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:36:10,475: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:38:51,069: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:38:51,072: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:38:51,075: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:38:51,078: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:41:36,794: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:41:36,796: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:41:36,800: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:41:36,802: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:45:36,642: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:45:36,645: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:45:36,649: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:45:36,651: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:55:04,681: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:55:04,685: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:55:04,690: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:55:04,692: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:55:50,281: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:55:50,284: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:55:50,290: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:55:50,292: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:59:33,872: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-08 22:59:33,875: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-08 22:59:33,881: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-08 22:59:33,883: INFO: common: Created direectories at: artifacts]
[2025-04-08 22:59:33,885: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 05:48:20,372: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 05:48:20,387: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 05:48:20,392: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 05:48:20,394: INFO: common: Created direectories at: artifacts]
[2025-04-09 05:48:20,396: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 05:59:42,116: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 05:59:42,116: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 05:59:42,116: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 05:59:42,116: INFO: common: Created direectories at: artifacts]
[2025-04-09 05:59:42,128: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 06:00:36,075: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 06:00:36,075: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 06:00:36,075: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 06:00:36,075: INFO: common: Created direectories at: artifacts]
[2025-04-09 06:00:36,075: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 08:08:32,882: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:08:32,902: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:08:32,916: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:08:32,918: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:08:32,919: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 08:08:34,398: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 08:15:41,501: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:15:41,504: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:15:41,506: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:15:41,506: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:15:41,506: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 08:15:41,939: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 08:46:24,886: INFO: main: Welcome to our cutome logging data science project]
[2025-04-09 08:46:24,907: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-09 08:46:24,911: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:46:24,912: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:46:24,914: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:46:24,915: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:46:24,916: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-09 08:46:24,916: INFO: data_ingestion: File already exists]
[2025-04-09 08:46:24,967: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:46:24,967: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-09 08:46:24,970: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:46:24,971: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:46:24,975: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:46:24,976: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:46:24,976: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-09 08:46:24,992: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:46:24,992: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-09 08:46:25,002: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:46:25,003: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:46:25,005: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:46:25,006: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:46:25,007: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-09 08:46:25,071: INFO: data_transformation: Splited data into training and test sets]
[2025-04-09 08:46:25,072: INFO: data_transformation: (1199, 12)]
[2025-04-09 08:46:25,072: INFO: data_transformation: (400, 12)]
[2025-04-09 08:46:25,073: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:46:25,073: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-09 08:46:25,076: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:46:25,077: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:46:25,080: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:46:25,081: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:46:25,081: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-09 08:46:25,159: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:46:25,159: INFO: main: >>>>>>>>>> Stage Model Evaluation Stage stated>>>>>>>>>>>]
[2025-04-09 08:46:25,162: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:46:25,163: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:46:25,165: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:46:25,166: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:46:25,166: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 08:46:26,427: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 08:46:31,693: INFO: main: >>>>>>>> Stage Model Evaluation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:49:07,691: INFO: main: Welcome to our cutome logging data science project]
[2025-04-09 08:49:07,691: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-09 08:49:07,695: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:49:07,696: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:49:07,699: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:49:07,700: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:49:07,700: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-09 08:49:07,701: INFO: data_ingestion: File already exists]
[2025-04-09 08:49:07,704: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:49:07,705: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-09 08:49:07,708: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:49:07,708: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:49:07,715: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:49:07,717: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:49:07,718: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-09 08:49:07,743: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:49:07,743: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-09 08:49:07,752: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:49:07,755: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:49:07,757: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:49:07,758: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:49:07,759: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-09 08:49:07,785: INFO: data_transformation: Splited data into training and test sets]
[2025-04-09 08:49:07,785: INFO: data_transformation: (1199, 12)]
[2025-04-09 08:49:07,786: INFO: data_transformation: (400, 12)]
[2025-04-09 08:49:07,786: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:49:07,786: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-09 08:49:07,790: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:49:07,791: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:49:07,793: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:49:07,793: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:49:07,794: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-09 08:49:07,824: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 08:49:07,824: INFO: main: >>>>>>>>>> Stage Model Evaluation Stage stated>>>>>>>>>>>]
[2025-04-09 08:49:07,827: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 08:49:07,828: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 08:49:07,831: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 08:49:07,831: INFO: common: Created direectories at: artifacts]
[2025-04-09 08:49:07,832: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 08:49:08,958: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 08:49:21,758: INFO: main: >>>>>>>> Stage Model Evaluation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:32:54,163: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.106:8080]
[2025-04-09 12:32:54,171: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-04-09 12:33:25,712: ERROR: app: Exception on / [GET]]
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\app.py", line 11, in homepage
    return render_template("index.htnl")
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.htnl
[2025-04-09 12:33:25,717: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:33:25] "[35m[1mGET / HTTP/1.1[0m" 500 -]
[2025-04-09 12:33:25,942: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:33:25] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-04-09 12:33:28,184: ERROR: app: Exception on / [GET]]
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\app.py", line 11, in homepage
    return render_template("index.htnl")
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.htnl
[2025-04-09 12:33:28,187: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:33:28] "[35m[1mGET / HTTP/1.1[0m" 500 -]
[2025-04-09 12:33:28,809: ERROR: app: Exception on / [GET]]
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\app.py", line 11, in homepage
    return render_template("index.htnl")
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.htnl
[2025-04-09 12:33:28,809: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:33:28] "[35m[1mGET / HTTP/1.1[0m" 500 -]
[2025-04-09 12:33:49,286: ERROR: app: Exception on / [GET]]
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\app.py", line 11, in homepage
    return render_template("index.htnl")
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.htnl
[2025-04-09 12:33:49,286: INFO: _internal: 192.168.0.106 - - [09/Apr/2025 12:33:49] "[35m[1mGET / HTTP/1.1[0m" 500 -]
[2025-04-09 12:33:49,445: INFO: _internal: 192.168.0.106 - - [09/Apr/2025 12:33:49] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-04-09 12:33:51,221: ERROR: app: Exception on / [GET]]
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\app.py", line 11, in homepage
    return render_template("index.htnl")
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.htnl
[2025-04-09 12:33:51,221: INFO: _internal: 192.168.0.106 - - [09/Apr/2025 12:33:51] "[35m[1mGET / HTTP/1.1[0m" 500 -]
[2025-04-09 12:34:26,375: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.106:8080]
[2025-04-09 12:34:26,375: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-04-09 12:34:31,662: ERROR: app: Exception on / [GET]]
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\app.py", line 11, in homepage
    return render_template("index.htnl")
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.htnl
[2025-04-09 12:34:31,666: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:34:31] "[35m[1mGET / HTTP/1.1[0m" 500 -]
[2025-04-09 12:34:31,744: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:34:31] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-04-09 12:34:33,942: ERROR: app: Exception on / [GET]]
Traceback (most recent call last):
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\app.py", line 11, in homepage
    return render_template("index.htnl")
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "D:\MLOPS_Cource\MLFLOW_Starter\datascienceproject\venv\lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.htnl
[2025-04-09 12:34:33,955: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:34:33] "[35m[1mGET / HTTP/1.1[0m" 500 -]
[2025-04-09 12:38:04,322: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.106:8080]
[2025-04-09 12:38:04,323: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-04-09 12:38:08,721: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:38:08] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:38:08,764: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:38:08] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:39:29,108: INFO: main: Welcome to our cutome logging data science project]
[2025-04-09 12:39:29,109: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-09 12:39:29,112: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:39:29,126: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:39:29,136: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:39:29,137: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:39:29,138: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-09 12:39:29,138: INFO: data_ingestion: File already exists]
[2025-04-09 12:39:29,183: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:39:29,184: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-09 12:39:29,187: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:39:29,188: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:39:29,191: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:39:29,192: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:39:29,192: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-09 12:39:29,243: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:39:29,244: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-09 12:39:29,248: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:39:29,249: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:39:29,251: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:39:29,252: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:39:29,252: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-09 12:39:29,353: INFO: data_transformation: Splited data into training and test sets]
[2025-04-09 12:39:29,353: INFO: data_transformation: (1199, 12)]
[2025-04-09 12:39:29,353: INFO: data_transformation: (400, 12)]
[2025-04-09 12:39:29,354: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:39:29,354: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-09 12:39:29,359: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:39:29,360: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:39:29,362: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:39:29,363: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:39:29,363: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-09 12:39:29,463: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:39:29,463: INFO: main: >>>>>>>>>> Stage Model Evaluation Stage stated>>>>>>>>>>>]
[2025-04-09 12:39:29,466: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:39:29,467: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:39:29,469: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:39:29,470: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:39:29,470: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 12:39:30,888: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 12:39:41,647: INFO: main: >>>>>>>> Stage Model Evaluation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:39:42,071: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:39:42] "GET /train HTTP/1.1" 200 -]
[2025-04-09 12:40:24,735: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:40:24] "POST /predict HTTP/1.1" 200 -]
[2025-04-09 12:42:52,740: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.106:8080]
[2025-04-09 12:42:52,741: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-04-09 12:42:57,187: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:42:57] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:42:57,223: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:42:57] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:43:08,512: INFO: main: Welcome to our cutome logging data science project]
[2025-04-09 12:43:08,512: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-09 12:43:08,512: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:43:08,528: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:43:08,532: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:43:08,533: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:43:08,533: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-09 12:43:08,534: INFO: data_ingestion: File already exists]
[2025-04-09 12:43:08,536: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:43:08,536: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-09 12:43:08,540: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:43:08,541: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:43:08,543: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:43:08,543: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:43:08,543: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-09 12:43:08,560: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:43:08,560: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-09 12:43:08,565: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:43:08,565: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:43:08,569: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:43:08,569: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:43:08,569: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-09 12:43:08,587: INFO: data_transformation: Splited data into training and test sets]
[2025-04-09 12:43:08,587: INFO: data_transformation: (1199, 12)]
[2025-04-09 12:43:08,587: INFO: data_transformation: (400, 12)]
[2025-04-09 12:43:08,587: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:43:08,587: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-09 12:43:08,592: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:43:08,594: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:43:08,598: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:43:08,599: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:43:08,600: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-09 12:43:08,629: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:43:08,630: INFO: main: >>>>>>>>>> Stage Model Evaluation Stage stated>>>>>>>>>>>]
[2025-04-09 12:43:08,634: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:43:08,635: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:43:08,638: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:43:08,638: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:43:08,639: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 12:43:09,312: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 12:43:14,167: INFO: main: >>>>>>>> Stage Model Evaluation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:43:14,563: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:43:14] "GET /train HTTP/1.1" 200 -]
[2025-04-09 12:43:56,975: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:43:56] "POST /predict HTTP/1.1" 200 -]
[2025-04-09 12:47:48,477: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.106:8080]
[2025-04-09 12:47:48,477: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-04-09 12:47:53,391: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:47:53] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:47:53,438: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:47:53] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:47:59,593: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:47:59] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:47:59,606: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:47:59] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:48:15,437: INFO: main: Welcome to our cutome logging data science project]
[2025-04-09 12:48:15,438: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-09 12:48:15,441: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:48:15,442: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:48:15,445: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:48:15,446: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:48:15,446: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-09 12:48:15,447: INFO: data_ingestion: File already exists]
[2025-04-09 12:48:15,449: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:48:15,449: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-09 12:48:15,451: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:48:15,451: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:48:15,457: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:48:15,457: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:48:15,457: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-09 12:48:15,472: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:48:15,473: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-09 12:48:15,477: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:48:15,479: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:48:15,481: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:48:15,482: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:48:15,483: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-09 12:48:15,501: INFO: data_transformation: Splited data into training and test sets]
[2025-04-09 12:48:15,501: INFO: data_transformation: (1199, 12)]
[2025-04-09 12:48:15,501: INFO: data_transformation: (400, 12)]
[2025-04-09 12:48:15,502: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:48:15,503: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-09 12:48:15,504: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:48:15,504: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:48:15,509: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:48:15,509: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:48:15,509: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-09 12:48:15,540: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:48:15,541: INFO: main: >>>>>>>>>> Stage Model Evaluation Stage stated>>>>>>>>>>>]
[2025-04-09 12:48:15,544: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:48:15,545: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:48:15,548: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:48:15,549: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:48:15,549: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 12:48:16,213: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 12:48:21,062: INFO: main: >>>>>>>> Stage Model Evaluation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:48:21,488: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:48:21] "GET /train HTTP/1.1" 200 -]
[2025-04-09 12:48:43,856: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:48:43] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:48:43,860: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:48:43] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:48:51,102: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:48:51] "POST /predict HTTP/1.1" 200 -]
[2025-04-09 12:54:40,255: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.106:8080]
[2025-04-09 12:54:40,255: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-04-09 12:54:45,645: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:54:45] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:54:45,694: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:54:45] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:55:09,433: INFO: main: Welcome to our cutome logging data science project]
[2025-04-09 12:55:09,434: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-09 12:55:09,437: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:55:09,438: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:55:09,441: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:55:09,442: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:55:09,443: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-09 12:55:09,443: INFO: data_ingestion: File already exists]
[2025-04-09 12:55:09,446: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:55:09,447: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-09 12:55:09,450: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:55:09,451: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:55:09,453: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:55:09,454: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:55:09,454: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-09 12:55:09,461: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:55:09,461: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-09 12:55:09,478: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:55:09,480: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:55:09,482: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:55:09,482: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:55:09,483: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-09 12:55:09,500: INFO: data_transformation: Splited data into training and test sets]
[2025-04-09 12:55:09,500: INFO: data_transformation: (1199, 12)]
[2025-04-09 12:55:09,500: INFO: data_transformation: (400, 12)]
[2025-04-09 12:55:09,501: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:55:09,501: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-09 12:55:09,504: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:55:09,505: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:55:09,508: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:55:09,509: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:55:09,510: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-09 12:55:09,531: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:55:09,531: INFO: main: >>>>>>>>>> Stage Model Evaluation Stage stated>>>>>>>>>>>]
[2025-04-09 12:55:09,543: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:55:09,544: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:55:09,545: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:55:09,546: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:55:09,546: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 12:55:10,230: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 12:55:15,210: INFO: main: >>>>>>>> Stage Model Evaluation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:55:15,623: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:55:15] "GET /train HTTP/1.1" 200 -]
[2025-04-09 12:55:38,973: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:55:38] "POST /predict HTTP/1.1" 200 -]
[2025-04-09 12:56:55,929: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.106:8080]
[2025-04-09 12:56:55,930: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-04-09 12:56:59,794: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:56:59] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:56:59,842: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:56:59] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:57:02,467: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:57:02] "GET / HTTP/1.1" 200 -]
[2025-04-09 12:57:02,483: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:57:02] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-04-09 12:57:13,079: INFO: main: Welcome to our cutome logging data science project]
[2025-04-09 12:57:13,080: INFO: main: >>>>>>>>>> Stage Data Ingestion Stage stated>>>>>>>>>>>]
[2025-04-09 12:57:13,083: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:57:13,084: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:57:13,086: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:57:13,087: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:57:13,087: INFO: common: Created direectories at: artifacts/data_ingestion]
[2025-04-09 12:57:13,088: INFO: data_ingestion: File already exists]
[2025-04-09 12:57:13,091: INFO: main: >>>>>>>> Stage Data Ingestion Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:57:13,092: INFO: main: >>>>>>>>>> Stage Data Validation Stage stated>>>>>>>>>>>]
[2025-04-09 12:57:13,095: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:57:13,096: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:57:13,099: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:57:13,100: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:57:13,101: INFO: common: Created direectories at: artifacts/data_validation]
[2025-04-09 12:57:13,116: INFO: main: >>>>>>>> Stage Data Validation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:57:13,117: INFO: main: >>>>>>>>>> Stage Data Transformation Stage stated>>>>>>>>>>>]
[2025-04-09 12:57:13,120: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:57:13,120: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:57:13,127: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:57:13,127: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:57:13,128: INFO: common: Created direectories at: artifacts/data_transformation]
[2025-04-09 12:57:13,145: INFO: data_transformation: Splited data into training and test sets]
[2025-04-09 12:57:13,146: INFO: data_transformation: (1199, 12)]
[2025-04-09 12:57:13,146: INFO: data_transformation: (400, 12)]
[2025-04-09 12:57:13,147: INFO: main: >>>>>>>> Stage Data Transformation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:57:13,147: INFO: main: >>>>>>>>>> Stage Model Trainer Stage stated>>>>>>>>>>>]
[2025-04-09 12:57:13,150: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:57:13,151: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:57:13,154: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:57:13,155: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:57:13,155: INFO: common: Created direectories at: artifacts/model_trainer]
[2025-04-09 12:57:13,184: INFO: main: >>>>>>>> Stage Model Trainer Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:57:13,184: INFO: main: >>>>>>>>>> Stage Model Evaluation Stage stated>>>>>>>>>>>]
[2025-04-09 12:57:13,189: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-04-09 12:57:13,190: INFO: common: yaml file: params.yaml loaded successfully]
[2025-04-09 12:57:13,191: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-04-09 12:57:13,192: INFO: common: Created direectories at: artifacts]
[2025-04-09 12:57:13,192: INFO: common: Created direectories at: artifacts/model_evaluation]
[2025-04-09 12:57:13,852: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-04-09 12:57:18,656: INFO: main: >>>>>>>> Stage Model Evaluation Stage completed >>>>><<<<<<<

x===========x]
[2025-04-09 12:57:19,066: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:57:19] "GET /train HTTP/1.1" 200 -]
[2025-04-09 12:57:30,715: INFO: _internal: 127.0.0.1 - - [09/Apr/2025 12:57:30] "POST /predict HTTP/1.1" 200 -]
